docs/en/wallet-solution.rst

```diff
@@ -13,38 +13,80 @@ such as Credential Issuers and Relying Parties.
  - The Wallet Instance MUST be compatible and functional on both Android and iOS operating systems and available on the Play Store and App Store, respectively.
  - The Wallet Instance MUST provide a mechanism to verify the User's actual possession and full control of their personal device.
 
+Wallet Attestation Requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Wallet Attestation contains information regarding the security level of the device hosting the Wallet Instance.
+It primarily certifies the **authenticity**, **integrity**, **security**, **privacy**, and **trustworthiness** of a particular Wallet Instance.
+
+.. figure:: ../../images/static_view_wallet_instance_attestation.svg
+   :name: Wallet Solution Schema
+   :alt: The image illustrates the containment of Wallet Provider and Wallet Instances within the Wallet Solution, managed by the Wallet Provider.
+   :target: https://www.plantuml.com/plantuml/uml/VP8nJyCm48Lt_ugdTexOCw22OCY0GAeGOsMSerWuliY-fEg_9mrEPTAqw-VtNLxEtaJHGRh6AMs40rRlaS8AEgAB533H3-qS2Tu2zxPEWSF8TcrYv-mJzTOGNfzVnXXJ0wKCDorxydAUjMNNYMMVpug9OTrR7i22LlaesXlADPiOraToZWyBsgCsF-JhtFhyGyZJgNlbXVR1oX5R2YSoUdQYEzrQO1seLcfUeGXs_ot5_VzqYM6lQlRXMz6hsTccIbGHhGu2_hhfP1tBwHuZqdOUH6WuEmrKIeqtNonvXhq4ThY3Dc9xBNJv_rSwQeyfawhcZsTPIpKLKuFYSa_JyOPytJNk5m00
+
+
+The requirements for the Wallet Attestation are defined below:
+
+- The Wallet Attestation MUST contain a Wallet Instance public key.
+- The Wallet Attestation MUST use the signed JSON Web Token (JWT) format;
+- The Wallet Attestation MUST provide all the relevant information to attest to the **integrity** and **security** of the device where the Wallet Instance is installed.
+- The Wallet Attestation MUST be signed by the Wallet Provider that has authority over and is the owner of the Wallet Solution, as specified by the overseeing registration authority. This ensures that the Wallet Attestation uniquely links the Wallet Provider to this particular Wallet Instance.
+- The Wallet Provider MUST ensure the integrity, authenticity, and genuineness of the Wallet Instance, preventing any attempts at manipulation or falsification by unauthorized third parties. The Wallet Provider MUST also verify the Wallet Instance using the App Store vendor's API, such as the *Play Integrity API* for Android and *DeviceCheck* for iOS. These services are defined in this specification as **Device Integrity Service (DIS)**.
+- The Wallet Attestation MUST have a mechanism in place for revoking the Wallet Instance, allowing the Wallet Provider to terminate service for a specific instance at any time.
+- The Wallet Attestation MUST be securely bound to the Wallet Instance's ephemeral public key.
+- The Wallet Attestation MAY be used multiple times during its validity period, allowing for repeated authentication and authorization without the need to request new attestations with each interaction.
```

Comment:
privacy concern.

please, add a consideration like "However, it is RECOMMENDED for the Wallet Instances to not use the same ..."